Lib/test/test_decimal.py

```diff
@@ -1089,6 +1089,15 @@ def test_formatting(self):
             ('07_', '1234.56', '1_234.56'),
             ('_', '1.23456789', '1.23456789'),
             ('_%', '123.456789', '12_345.6789%'),
+            # and now for something completely different...
+            ('.,', '1.23456789', '1.234,567,89'),
+            ('._', '1.23456789', '1.234_567_89'),
+            ('.6_f', '1.23456789', '1.234_568'),
+            (',._%', '123.456789', '12,345.678_9%'),
```

Comment:
Test also the thousands separator for fraction part only, without separator for integer part.